source/agora/common/BitField.d

```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+
+    Contains a BitField implementation based on Ocean's BitArray struct.
+    The choice of Ocean's over Phobos' implementation of BitArray is
+    because the Phobos implementation requires the backing store length to be
+    a multiple of size_t, which makes it platform-specific and hard to code for.
```

Comment:
Maybe mention that this structure cannot be platform specific because it is stored in the block. Platform specific behavior is not bad if it's not used in data that is exchanged. I'd guess the platform-specifism of Phobos is to make it more performant.

Reply:
> I'd guess the platform-specifism of Phobos is to make it more performant.

They probably used size_t without giving it a second thought.